### DISCUSSION TITLE
Infomation screen

### DISCUSSION THREAD

**Discussion by Suggy50** [FYSETC/FYSETC-E4#7] [Ideas]
How to add screen for monitoring purposes

**Comment by GerogeFu** [comment]
This board only support serial screen, there is serial interface 
![image](https://user-images.githubusercontent.com/7496375/101603606-c86c7f80-3a3a-11eb-9f40-2b8349d1046c.png)

**Reply by Suggy50** [reply]
I'm aware it will take a serial screen but recommendations on which screens will work and if any firmware will be needed for the screens would be a big help

**Reply by GerogeFu** [reply]
Actually , we are making a serial screen for it if you can wait, you can search `3D print TFT35` in aliexpress if you really want to have one now. And you can follow that serial screen instruction. By the way, E4 board and the serial screen are seperated parts, serial screen is not  drived by E4.

**Reply by Suggy50** [reply]
Do you have a time scale for release?

**Reply by OriginaldCrypt** [reply]
Hi, does this serial interface share the pin connection with TMC 2209 UART control? I mean, does enabling a serial LCD mean that TMC 2209 UART mode is disabled?

**Reply by TextileTopher** [reply]
I’m also curious about Crypts question. I’m having trouble getting a second serial working (hardware or software). 

This is a really nice board! Congrats to the designer!

**Comment by OriginaldCrypt** [comment]
Hi, given that this is a serial communication, and assuming that you refer to a display with GCode sending capabilities, could it be useful to make a serial test with an Arduino or USB-2-Serial to test this approach?

**Comment by OriginaldCrypt** [comment]
See here for my approach to use a serial connection in Fysetc E4 in a project:

https://github.com/FYSETC/FYSETC-E4/issues/53#issuecomment-2522887573

I hope to provide clarity on something because the Fysetc documentation is very bad and inaccurate.
